ogr/ogrsf_frmts/flatgeobuf/geometrywriter.cpp

```diff
@@ -91,6 +91,9 @@ uint32_t GeometryWriter::writeSimpleCurve(const OGRSimpleCurve *sc)
 
 void GeometryWriter::writeMultiLineString(const OGRMultiLineString *mls)
 {
+    // NOTE: should not write ends if less than 2 parts
```

Comment:
hum that seems wrong to me. This will not even write the geometry at all if it made of one part... unless I'm missing something. The loop below doesn't only update m_ends: writeSimpleCurve() sets the x/y/z/m arrays

Reply:
ah yes, I was clearly confused, thanks.